### DISCUSSION TITLE
[Feature] Search box doesn't generate exact result.

### DISCUSSION THREAD

**Discussion by adsingh14** [simple-icons/simple-icons#6718] [General]
### Test case:

- Open this link: https://simpleicons.org/?q=steel

**OR**

- Type '**steel**' in searchbox

![image](https://user-images.githubusercontent.com/35252877/136998355-d313b91a-d3a2-4013-83aa-8b39d485c2aa.png)

**Comment by jorgeamadosoria** [comment]
this is a bug report for the website repo. 
If you go there, you will find that there is an issue open already to improve the fuzzy search code.
Maybe you can go to that issue (https://github.com/simple-icons/simple-icons-website/issues/36) and add your example there if it applies?

**Reply by adsingh14** [reply]
They've mentioned it already. It'd be great if we get at least two examples below search box or a info icon near search box, telling about 'fuzzy search'.

**Reply by jorgeamadosoria** [reply]
do you think you can make the placeholder message reflect what you mean, without adding new elements to the page?
If you want to add this, I suggest you either:
- add a comment on the issue for the fuzzy search
- Open a new issue on the website with your proposal, so we can comment and discuss it.

I don't particularly prefer any over the others, feel free to choose the one you think it's best.

**Reply by adsingh14** [reply]
I don't think we need to discuss about this further. I'm satisfied with the issue reply. Someone will either read this or the issue eventually.

It'd be great if you can terminate this discussion. :)

**Comment by adamrusted** [comment]
Going to lock this conversation, as the issue seems to have been resolved in a recent update.
